scripts/dev.mk

```diff
@@ -4,9 +4,19 @@ db/verify-global-migration:
 	./gradlew :persistence:verifyCommonMainGlobalDatabaseMigration
 
 db/verify-user-migration:
+	rm persistence/src/commonMain/db_user/schemas/2.db
+	for i in {1..33}; \
+	do \
+		rm persistence/src/commonMain/db_user/migrations/$$i.sqm; \
+	done
```

Comment:
Could you please just add a comment here saying why this is here. So if we ever forget and someone else finds it 😅